extensions/pvaPublish/webpack.config.js

```diff
@@ -38,6 +38,9 @@ function getWebpackConfigs() {
         extensions: ['.js', '.ts', '.tsx'],
       },
       plugins: [],
+      node: {
+        fs: 'empty',
```

Comment:
what is this for?

Reply:
The `botbuilder-lg` can target both node env and browser env, to build it correctly in browser env, any project imports `botbuilder-lg` needs this line to instruct webpack where to resolve `node`. Otherwise, there will be a build error `fs not empty`. https://github.com/webpack-contrib/css-loader/issues/447#issuecomment-285598881

I add such config in this PR because `@bfc/extension-client` starts to depend on `botbuilder-lg`, this change led to a build failure in pvaPublish (it depends on `@bfc/extension-client`).